matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/data/LoginFlowResponse.kt

```diff
@@ -42,7 +42,7 @@ internal data class LoginFlow(
          * the client can show a button for each of the supported providers
          * See MSC #2858
          */
-        @Json(name = "org.matrix.msc2858.identity_providers")
+        @Json(name = "identity_providers")
```

Comment:
Shouldn't we add a field for `identity_providers`, and use it first, or take `org.matrix.msc2858.identity_providers` if present?

Reply:
synapse is already using stable id since synapse 1.30, so we can just drop

Reply:
ok, fine